.github/workflows/lighty-rnc-app/simulator/Dockerfile

```diff
@@ -1,24 +1,24 @@
 ARG SIMULATOR_VERSION="21.0.0"
 
-FROM alpine:3.21.2 as clone
+FROM alpine:3.21.2 AS clone
 
 ARG SIMULATOR_VERSION
 RUN apk add git
 WORKDIR /netconf-simulator
 RUN git clone https://github.com/PANTHEONtech/lighty-netconf-simulator.git -b $SIMULATOR_VERSION
 
-FROM maven:3.9-eclipse-temurin-21-alpine as build
+FROM maven:3.9-eclipse-temurin-21-alpine AS build
 ARG SIMULATOR_VERSION
 WORKDIR /lighty-netconf-simulator
-COPY --from=clone /netconf-simulator/lighty-netconf-simulator /lighty-netconf-simulator
+COPY --from=CLONE /netconf-simulator/lighty-netconf-simulator /lighty-netconf-simulator
 RUN mvn -B install -DskipTests
 
 FROM eclipse-temurin:21-jre-alpine
 ARG SIMULATOR_VERSION
 WORKDIR /lighty-netconf-simulator
-COPY --from=build /lighty-netconf-simulator/examples/devices/lighty-network-topology-device/target/ /lighty-netconf-simulator/target
+COPY --from=BUILD /lighty-netconf-simulator/examples/devices/lighty-network-topology-device/target/ /lighty-netconf-simulator/target
 
 EXPOSE 17380
 
 ENV APP_PATH="/lighty-netconf-simulator/target/lighty-network-topology-device-$SIMULATOR_VERSION.jar"
```

Comment:
APP_PATH is not used

Reply:
Resolved:
https://github.com/PANTHEONtech/lighty/pull/2209/files#diff-cbd0a08174d6cde84c678cdf886cff515d42caf7c9fe1a6f1eee79d45cfead77R24